main/templates/main/edit_event.html

```diff
@@ -0,0 +1,192 @@
+{% extends 'main/general.html' %}
+
+{% block title %} Edit Event {% endblock %}
+
+{% block css %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'styles/main/edit_event_styles.css' %}">
+
+{% endblock %}
+
+{% block maincontent %}
+<h2>Edit Event</h2>
+
+<form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Save Changes</button>
+</form>
+{% endblock %}
+
+{% block style %}
+<style>
```

Comment:
Styles can be moved to the correct file (there are quite a few of them) if it does not break the styles in the main html file

Reply:
I agree with you. I will transfer all the styles to the css file.